### PR TITLE
[image_picker] Update image_picker to 1.0.1

### PR DESCRIPTION
**Pull request by Swanseo0**
Update package dependencies.
- Add `ImagePicker.pickMedia` and `ImagePicker.supportsImageSource` method.
- Update example

### CODE REVIEW

File: packages/image_picker/example/pubspec.yaml [hunk 1]
```diff
@@ -9,10 +9,11 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  image_picker: ^0.8.6
+  image_picker: ^1.0.1
   image_picker_tizen:
     path: ../
-  video_player: ">=2.1.4 <2.7.0"
+  video_player: ^2.4.2
+  video_player_tizen: ^2.4.7
```
Comment (JSUYA) [inline packages/image_picker/example/pubspec.yaml:17]: You need to add mime package dependency.
https://github.com/flutter/packages/blob/main/packages/image_picker/image_picker/example/pubspec.yaml#L21C15-L21C15

Comment (Swanseo0) [reply packages/image_picker/example/pubspec.yaml:18]: Added mime package dependency. thank you.

**Review by JSUYA — APPROVED** [review]
LGTM

File: packages/image_picker/tizen/src/image_picker_tizen_plugin.cc [hunk 2]
```diff
@@ -112,10 +112,39 @@ class ImagePickerTizenPlugin : public flutter::Plugin {
...
+
+      ImageSource source = ImageSource(source_index);
+      if (source == ImageSource::kCamera) {
+        // TODO: we need to check this feature after webcam is prepared
+        SendErrorResult("Not supported", "Not supported on this device.");
+      } else if (source == ImageSource::kGallery) {
+        multi_image_ = multiple;
+        PickContent("image/*, video/*");
+      } else {
+        SendErrorResult("Invalid arguments", "Invalid image source.");
```
[the remark this answers is not in the document]
Comment (Swanseo0) [reply packages/image_picker/tizen/src/image_picker_tizen_plugin.cc]: Applied. thank you!

**Pull request merged**